### ISSUE TITLE
OS not compatible with Uber Driver or Dasher apps

### ISSUE THREAD

**Issue by ruvilonix** [GrapheneOS/os-issue-tracker#2317]
I deliver and do rideshare for Uber and Doordash. The apps to work those platforms were basically non-functional on GrapheneOS. I think Dasher never worked, and Uber was hit or miss. They didn't give any specific errors like "Custom ROM", they were just very buggy. 

I was using my old Pixel 3a with stock Android for a while for these apps, using a Wifi hotspot from the 6a with Graphene, but it was very inconvenient. 

I've now put stock Android back on the 6a so that I can use it for work. I can't create bug reports because it's no longer GrapheneOS, but I wanted to report this issue, and hopefully someone can figure out how to fix the issue.

I miss GrapheneOS and feel so dirty and exposed on Android.

**Comment by thestinger** [comment]
We've heard that the apps do work with sandboxed Google Play. You'll need to be specific about what doesn't work. There will be faster location detection if you've configured the network location service as described in our usage guide at https://grapheneos.org/usage#sandboxed-google-play-configuration.

**Comment by ruvilonix** [comment]
I'm not sure if I did the network thing, but I definitely had sandboxed Google Play.

With Dasher, one of the main bugs is that is wouldn't go past the login screen, just refreshed with blank fields.

With Uber, it was the "Go" online button, it would say there's a network issue, and to wait a few minutes, but it wouldn't start working even after long waits. So maybe it's related to the network settings you mentioned. Also Uber didn't provide voice directions even when I did get online, and the setting showed as on.
